thread/stack-allocator.cpp

```diff
@@ -37,7 +37,11 @@ class PooledStackAllocator {
         __builtin_ffsl(MAX_ALLOCATION_SIZE / MIN_ALLOCATION_SIZE);
 
 public:
-    PooledStackAllocator() {}
+    PooledStackAllocator() {
+        for (size_t i = 0; i < N_SLOTS; i++) {
+            slots[i].slotsize = MIN_ALLOCATION_SIZE << i;
```

Comment:
This is  constant to a slot, why should we store it in a variable?

Reply:
`Slot::get` needs allocation with size, and it likes a property of a slot.